File: DevExpress-Demo/DevExpress.SalesDemo.Wpf/CS/DevExpress.SalesDemo.Wpf/Helpers/ResourceImageHelper.cs
using DevExpress.SalesDemo.Model;
using System.Windows.Media.Imaging;

namespace DevExpress.SalesDemo.Wpf.Helpers {
    public static class ResourceImageHelper {
        public static BitmapImage GetResourceImage(string image) {
            BitmapImage res = new BitmapImage();
            res.BeginInit();
            res.StreamSource = ModelAssemblyHelper.GetResourceStream(image);
            res.EndInit();
            res.Freeze();
            return res;
        }
    }
}

File: DevExpress-Demo/DevExpress.SalesDemo.Wpf/VB/DevExpress.SalesDemo.Wpf/Helpers/ResourceImageHelper.vb
Imports DevExpress.SalesDemo.Model
Imports System.Windows.Media.Imaging

Namespace DevExpress.SalesDemo.Wpf.Helpers

    Public Module ResourceImageHelper

        Public Function GetResourceImage(ByVal image As String) As BitmapImage
            Dim res As BitmapImage = New BitmapImage()
            res.BeginInit()
            res.StreamSource = ModelAssemblyHelper.GetResourceStream(image)
            res.EndInit()
            res.Freeze()
            Return res
        End Function
    End Module
End Namespace
